src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -48,6 +50,24 @@ type InitData struct {
 	Data      map[string]string `toml:"data,omitempty"`
 }
 
+type ServerConfig struct {
```

Comment:
Should this be CaaConfig ? `ServerConfig` is overloaded.

Reply:
The `ServerConfig` type pre-dates this PR. 
It was moved from `src/cloud-api-adaptor/pkg/adaptor/server.go`. 

If we want to change it, we should do that in a separate PR.

Reply:
Yeah it makes sense.